ui/src/main/js/distribution/DistributionConfigurationForm.js

```diff
@@ -163,8 +170,16 @@ const DistributionConfigurationForm = ({
         const channelKey = FieldModelUtilities.getFieldModelSingleValue(channelModel, DISTRIBUTION_COMMON_FIELD_KEYS.channelName);
         const providerFound = FieldModelUtilities.hasValue(channelModel, DISTRIBUTION_COMMON_FIELD_KEYS.providerName);
         setProviderHasChannelName(true);
-        if (channelKey === EMAIL_INFO.key) {
+        if (channelKey === AZURE_INFO.key) {
```

Comment:
In a setting like this, do we prefer a switch over a bunch of if statements?

Reply:
Since a switch uses strict comparison we should change to that here.